docs/ingestion/input-sources.md

```diff
@@ -1147,11 +1147,12 @@ To use the Delta Lake input source, load the extension [`druid-deltalake-extensi
 You can use the Delta input source to read data stored in a Delta Lake table. For a given table, the input source scans
 the latest snapshot from the configured table. Druid ingests the underlying delta files from the table.
 
-| Property|Description|Required|
-|---------|-----------|--------|
-| type|Set this value to `delta`.|yes|
-| tablePath|The location of the Delta table.|yes|
-| filter|The JSON Object that filters data files within a snapshot.|no|
+| Property|Description| Default|Required |
+|---------|-----------|-----------------|
+|type|Set this value to `delta`.| None|yes|
+|tablePath|The location of the Delta table.|None|yes|
+|filter|The JSON Object that filters data files within a snapshot.|None|no|
+|snapshot|The JSON Object that specifies a Delta table snapshot to read from.|latest|no|
```

Comment:
does it have to be a `json`?

Reply:
we could try to parse the snapshot as a number first and if that fails, see if its equal to `latest`, otherwise give an error.

Reply:
Used JSON primarily for extensibility. For example, when we add snapshots by [timestamp](https://docs.delta.io/latest/api/java/kernel/index.html?io/delta/kernel/engine/Engine.html), which accpets epoch millis, parsing the value could lead to ambiguity.

Reply:
Flatenned this out based on our offline discussion